### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by lfarrell**
https://jira.lib.unc.edu/browse/BXC-2797
Depends on https://gitlab.lib.unc.edu/cdr/boxc-ansible/-/merge_requests/81

### CODE REVIEW

File: server/test/arrow/xml-test.js [hunk 1]
```diff
@@ -125,6 +125,6 @@ describe('XML', function() {
 
     let xml = new XML(root);
 
-    equal(xml.render().toString(), '<values property="[object Object]" childAttrWithChildren="[object Object]" childAttrWithValueProperty="[object Object]">[object Object]</values>');
+    equal(xml.render().toString(), '<values property="[object Object]" childAttrWithChildren="[object Object]" childAttrWithValueProperty="[object Object]"><a>1</a>[object Object]</values>');
```
Comment (bbpennel) [inline server/test/arrow/xml-test.js:128]: This change seems a bit random, is it really now adding `<a>1</a>` into the xml?

**Review by bbpennel — COMMENTED** [review]
It seems like many of the changes are whitespace changes, or switching backticks to quotes, were those both required or was the IDE having a good time? Anyways, it doesn't look like too scary of an update. Good work.

We should probably wait until we can test this on a server before merging

**Comment by lfarrell** [comment]
There were a lot of tabs/spaces on empty lines. I think that was the major reason for the indent changes.

File: server/test/arrow/xml-test.js (hunk 1 above)
Comment (lfarrell) [reply server/test/arrow/xml-test.js:128]: It seemed like updating the underlying xml libraries changes what's rendered

**Pull request merged**